packages/react-router/src/ReactRouter/StackManager.tsx

```diff
@@ -197,11 +216,16 @@ export class StackManager extends React.PureComponent<StackManagerProps, StackMa
     const canStart = () => {
       const config = getConfig();
       const swipeEnabled = config && config.get('swipeBackEnabled', routerOutlet.mode === 'ios');
-      if (!swipeEnabled) { return false; }
+      if (!swipeEnabled) {
+        return false;
+      }
```

Comment:
Are these changes from prettier or from your built in IDE? If they are from the IDE, could we remove them?

Reply:
These changes are from prettier. The extension detects the local configuration: https://github.com/ionic-team/ionic-framework/blob/main/packages/react-router/.prettierrc.json and is formatting based off that.

Unfortunately the React packages do not actually run prettier as part of the lint step at all.

Reply:
Oh right! Never mind 👍